src/common/index.ts

```diff
@@ -15,19 +16,24 @@ export enum ViewMode { Gallery = 'gallery', List = 'list' }
 
 export const isList = (viewMode: string) => viewMode === ViewMode.List;
```

Comment:
With the introduction of the "List" class this helper function becomes a bit confusing. I think it could just be removed and the code inlined.